wrappers/javascript/anoncreds-react-native/cpp/turboModuleUtility.cpp

```diff
@@ -255,10 +258,13 @@ jsiToValue<FfiList_FfiCredentialProve>(jsi::Runtime &rt, jsi::Object &options,
       auto isPredicate = jsiToValue<int8_t>(rt, valueAsObject, "isPredicate");
       auto reveal = jsiToValue<int8_t>(rt, valueAsObject, "reveal");
 
+      char *ffiStr = new char[sizeof(referent)];
```

Comment:
Any specific reason for this change?

Reply:
Yes, it was not working if passed directly to the struct. Probably also related with that automatic wiping that is happening.